crates/optimism/payload/src/builder.rs

```diff
@@ -183,6 +183,15 @@ where
     });
     let base_fee = initialized_block_env.basefee.to::<u64>();
 
+    let is_holocene = chain_spec.is_fork_active_at_timestamp(
+        OptimismHardfork::Holocene,
+        attributes.payload_attributes.timestamp,
+    );
+
+    if is_holocene && attributes.eip_1559_params.is_none() {
```

Comment:
This needs to be lifted into the engine - see [the OptimismEngineValidator](https://github.com/paradigmxyz/reth/blob/main/crates/optimism/node/src/engine.rs#L138-L152)

Reply:
Added
https://github.com/paradigmxyz/reth/pull/11473/files#diff-c61b1dc2b86be8360bc0819742ae3d7a71810827532b244b5c05f5ff62a1f370R151-R160